Ghostscript.NET.PDFA3Converter.Samples/ZUGFeRD/ZUGFeRDImporter.cs

```diff
@@ -0,0 +1,731 @@
+        /// </summary>
+        //JAVA TO C# CONVERTER NOTE: Fields cannot have the same name as methods of the current type:
+        protected internal bool containsMeta_Conflict = false;
+        protected System.Xml.XmlDocument xmlDoc;
+        /// <summary>
+        /// map filenames of additional XML files to their contents
+        /// </summary>
+        private Dictionary<string, sbyte[]> additionalXMLs = new Dictionary<string, sbyte[]>();
+        /// <summary>
+        /// Raw XML form of the extracted data - may be directly obtained.
+        /// </summary>
+        private byte[] rawXML = null;
+        /// <summary>
+        /// XMP metadata
+        /// </summary>
+        private string xmpString = null; // XMP metadata
+        /// <summary>
+        /// parsed Document
+        /// </summary>
+        private XmlDocument document;
+
+        private readonly ILogger<ZUGFeRDImporter> _logger;
+
+
+        protected internal ZUGFeRDImporter()
+        {
+            //constructor for extending classes
+        }
+
+        public ZUGFeRDImporter(string pdfFilename)
+        {
+            using var loggerFactory = LoggerFactory.Create(builder =>
+        {
+            builder
+                .AddFilter("Microsoft", LogLevel.Warning)
+                .AddFilter("System", LogLevel.Warning)
+                .AddFilter("LoggingConsoleApp.Program", LogLevel.Debug);
+        });
+
+            if (!File.Exists(pdfFilename)) {
+                throw new Exception("File not found");
+            }
+
+            _logger = loggerFactory.CreateLogger<ZUGFeRDImporter>();
+            _logger.LogInformation("Example log message");
+            FromPDF(pdfFilename);
+            //FileStream fs = File.OpenRead(pdfFilename);
+            //		extractLowLevel(fs);
+        }
+
+
+        public ZUGFeRDImporter(Stream pdfStream)
+        {
+            try
+            {
+                extractLowLevel(pdfStream);
+            }
+            //JAVA TO C# CONVERTER WARNING: 'final' catch parameters are not available in C#:
+            //ORIGINAL LINE: catch (final java.io.IOException e)
+            catch (IOException e)
+            {
+                //JAVA TO C# CONVERTER WARNING: The .NET Type.FullName property will not always yield results identical to the Java Class.getName method:
+                //		_logger.LogError(e.Message);
+                //		throw new ZUGFeRDExportException(e);
+            }
+        }
+
+
+        /// <summary>
+        /// Extracts a ZUGFeRD invoice from a PDF document represented by an input stream. Errors are reported via exception handling.
+        /// </summary>
+        /// <param name="pdfStream"> a inputstream of a pdf file </param>
+        //JAVA TO C# CONVERTER WARNING: Method 'throws' clauses are not available in C#:
+        //ORIGINAL LINE: private void extractLowLevel(java.io.InputStream pdfStream) throws java.io.IOException
+        private void extractLowLevel(Stream pdfStream)
+        {
+
```

Comment:
need to implement that.

Reply:
In the original Java code, pdfbox is used:
https://github.com/ZUGFeRD/mustangproject/blob/4987505b6c975df21a8a1bc61ba26d0dde56596a/library/src/main/java/org/mustangproject/ZUGFeRD/ZUGFeRDInvoiceImporter.java#L28

The code where this pull request is based on, also contains an empty function:

https://github.com/ArtifexSoftware/Ghostscript.NET/blob/f1e9a4672a9b33646390732f6629c90642f6bce0/Ghostscript.NET/FacturX/ZUGFeRD/ZUGFeRDImporter.cs#L114

I can only propose to remove ZUGFeRDImporter unless you have some sample code to process pdf files that I could use as a basis.

Reply:
Hi @stephanstapel , if you could not implement that, please remove that.

Reply:
removed, unfortunately